extensions/amp-sidebar/1.0/test/validator-amp-sidebar.out

```diff
@@ -1,3 +1,3 @@
 FAIL
-amp-sidebar/1.0/test/validator-amp-sidebar.html:51:4 The tag 'amp-ad' may not appear as a descendant of tag 'amp-sidebar'. (see https://www.ampproject.org/docs/reference/components/amp-ad) [AMP_TAG_PROBLEM]
-amp-sidebar/1.0/test/validator-amp-sidebar.html:54:2 The attribute 'side' in tag 'amp-sidebar' is set to the invalid value 'center'. (see https://www.ampproject.org/docs/reference/components/amp-sidebar) [AMP_TAG_PROBLEM]
+amp-sidebar/1.0/test/validator-amp-sidebar.html:50:4 The tag 'amp-ad' may not appear as a descendant of tag 'amp-sidebar'. (see https://www.ampproject.org/docs/reference/components/amp-ad) [AMP_TAG_PROBLEM]
```

Comment:
I'm not sure I follow. This change is simply because the validator test file had the wrong line numbers. Most likely it was a line added right before commit.